.github/ISSUE_TEMPLATE/application.yml

```diff
@@ -0,0 +1,181 @@
+    label: Project Description
+    description: Provide a brief, 100-300 word description of the project that explains what it does and why its needed.
+    placeholder: Describe what it does and why it's needed.
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Org repo URL
+    description: Provide the URL of GitHub or Gitlab organization of the project. If no organization, provide "N/A".
+    placeholder: Add org URL here
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Project repo URL
+    description: Provide the URL of the primary source code for the project. If no primary repo exists, provide "N/A" and enumerate repos in the next question.
+    placeholder: Add code repo URL here
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Additional repos
+    description: Provide the URL any additional repos for this project if appropriate.
+    placeholder: Add additional repo URLs here
+  validations:
+    required: false
+- type: input
+  attributes:
+    label: Website URL
+    description: Provide the URL of the project's website. If the project has no website, please provide the primary repo URL again.
+    placeholder: Add website URL here
+  validations:
+    required: false
+- type: input
+  attributes:
+    label: Roadmap
+    description: Provide the URL of the project's roadmap.
+    placeholder: Add Roadmap URL here
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Roadmap context
+    description:  You may optionally provide any details of the roadmap you feel are required for clarity.
+  validations:
+    required: false
+- type: input
+  attributes:
+    label: Contributing Guide
+    description: Provide the URL of the project's contribution guide. If the guide is embedded in another file, please deep link to the correct header.
+    placeholder: Add Contributing guide URL here
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Code of Conduct (CoC)
+    description: Provide the URL of the project's code of conduct.
+    placeholder: Add CoC URL here
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Adopters
+    description: Provide the URL of the project's Adopters file. If no file exists, move on to the next question.
+    placeholder: Add Adopters URL here
+  validations:
+    required: false
+- type: input
+  attributes:
+    label: Contributing or Sponsoring Org
+    description: Provide the URL of the project's contributing or sponsoring company/organization. If no such company/organization exists move on to the next question.
+    placeholder: Add sponsoring company or org URL here
+  validations:
+    required: false
+- type: input
+  attributes:
+    label: Maintainers file
+    description: Provide the URL of the project's maintainers file.
+    placeholder: Add maintainer URL here
+  validations:
+    required: true
+- type: textarea
```

Comment:
This looks to be an errant type declaration. Removing line 103 should fix things.